src/d3d/nvapi_d3d_low_latency_device.h

```diff
@@ -26,21 +26,24 @@ namespace dxvk {
     class NvapiD3dLowLatencyDevice {
       public:
         static bool SupportsLowLatency(IUnknown* device);
+        static bool SupportsLowLatency(ID3D12CommandQueue* commandQueue);
         static bool LatencySleep(IUnknown* device);
         static bool SetLatencySleepMode(IUnknown* device, bool lowLatencyMode, bool lowLatencyBoost, uint32_t minimumIntervalUs);
         static bool GetLatencyInfo(IUnknown* device, D3D_LATENCY_RESULTS* latencyResults);
         static bool SetLatencyMarker(IUnknown* device, uint64_t frameID, uint32_t markerType);
+        static bool SetLatencyMarker(ID3D12CommandQueue* commandQueue, uint64_t frameID, uint32_t markerType);
```

Comment:
I was wondering if this shouldn't be called `SetAsyncFrameMarker` to better match Reflex API but I feel like it would be lying about the implementation. So I'm not sure which name would be preferable here.

Reply:
I think `SetLatencyMarker` is fine since we call this thing `LowLatencyDevice` and not `ReflexDevice`.